modules/sovrnBidAdapter.js

```diff
@@ -23,7 +23,9 @@ export const spec = {
    */
   buildRequests: function(bidReqs) {
     let sovrnImps = [];
+    let iv;
     utils._each(bidReqs, function (bid) {
+      iv = iv || utils.getBidIdParameter('iv', bid.params);
```

Comment:
So.. can we have multiple `iv` params sent into the adapter for each tag, but we only attach one to the request?

Reply:
right. so the conversation i had with the onetag guys was that if there is an iv param on a tag, it will be on all the tags and will have the same value, and that it will be up to them to ensure that is true

Reply:
Alright